javascript-conditionals/main.js

```diff
@@ -0,0 +1,465 @@
+    // )
+  } else if (genre === 'comedy') {
+    const comedyMovies = [
+      'Anchorman: The Legend of Ron Burgundy',
+      'Superbad',
+      'The Hangover',
+      'Dumb and Dumber',
+      'Bridesmaids',
+      'Step Brothers',
+      "Ferris Bueller's Day Off",
+      'The Grand Budapest Hotel',
+      'Mean Girls',
+      'Pineapple Express',
+      'Shaun of the Dead',
+      'Napoleon Dynamite',
+      'The 40-Year-Old Virgin',
+      'Wedding Crashers',
+      'Elf',
+      'Hot Fuzz',
+      'Airplane!',
+      'Austin Powers: International Man of Mystery',
+      'The Big Lebowski',
+      'Zoolander',
+      'Ted',
+      'Tropic Thunder',
+      'Old School',
+      'Pitch Perfect',
+      'Dodgeball: A True Underdog Story',
+      'Annie Hall',
+      'Caddyshack',
+      'The Princess Bride',
+      'Ghostbusters',
+      'Coming to America',
+      'Nacho Libre',
+      'The Pink Panther',
+      'Clueless',
+      "Ferris Bueller's Day Off",
+      'The Naked Gun: From the Files of Police Squad!',
+      "Wayne's World",
+      'Dazed and Confused',
+      'Office Space',
+      'Happy Gilmore',
+      'The Devil Wears Prada',
+      'Bruce Almighty',
+      'Meet the Parents',
+      'Neighbors',
+      'Borat',
+      "There's Something About Mary",
+      'Four Weddings and a Funeral',
+      'The Producers',
+      'Trading Places',
+      'Juno',
+      'Rushmore',
+    ];
+
+    return comedyMovies[Math.floor(Math.random() * comedyMovies.length)];
+  } else if (genre === 'musical') {
+    const musicalMovies = [
+      'The Shawshank Redemption',
+      'Forrest Gump',
+      'The Godfather',
+      "Schindler's List",
+      'Fight Club',
+      'Pulp Fiction',
+      'American Beauty',
+      'Good Will Hunting',
+      'The Green Mile',
+      'A Beautiful Mind',
+      'The Social Network',
+      'The Pursuit of Happyness',
+      'Requiem for a Dream',
+      'Eternal Sunshine of the Spotless Mind',
+      '12 Angry Men',
+      'The Departed',
+      'No Country for Old Men',
+      'The Wolf of Wall Street',
+      'Gladiator',
+      'Braveheart',
+      'Atonement',
+      'The Help',
+      'Manchester by the Sea',
+      'Spotlight',
+      'Million Dollar Baby',
+      'Dead Poets Society',
+      'Rain Man',
+      'American History X',
+      'La La Land',
+      "The King's Speech",
+      'Whiplash',
+      'The Theory of Everything',
+      'Black Swan',
+      'Gone Girl',
+      'Moonlight',
+      'Room',
+      'There Will Be Blood',
+      'Dallas Buyers Club',
+      'The Imitation Game',
+      "Boys Don't Cry",
+      'Fences',
+      'The Pianist',
+      'Scent of a Woman',
+      'Mystic River',
+      'A Star Is Born',
+      'Little Women',
+      'Ordinary People',
+      'Shakespeare in Love',
+      'The Bridges of Madison County',
+    ];
+
+    return musicalMovies[Math.floor(Math.random() * musicalMovies.length)];
+  }
+}
+
```

Comment:
As noted in the instructions, this `function` needs to be completed via a `switch` statement rather than using `else..if` statements